src/node/server/serverPluginCss.ts

```diff
@@ -133,7 +133,8 @@ export const cssPlugin: ServerPlugin = ({ root, app, watcher, resolver }) => {
       filename: resolver.requestToFile(ctx.path),
       scoped: false,
       modules: ctx.path.includes('.module'),
-      preprocessLang: ctx.path.replace(cssPreprocessLangRE, '$2') as any
+      preprocessLang: ctx.path.replace(cssPreprocessLangRE, '$2') as any,
+      preprocessOptions: ctx.config.cssPreprocessOptions
```

Comment:
The current implementation is not correct.

I believe this should be like this:

```ts
const preprocessLang = ctx.path.replace(cssPreprocessLangRE, '$2') as any
// ...

const result = await compileCss(root, ctx.path, {
  id: '',
  source: css,
  filename: resolver.requestToFile(ctx.path),
  scoped: false,
  modules: ctx.path.endsWith('.module.css'),
  preprocessLang,
  preprocessOptions: ctx.config.cssPreprocessOptions[preprocessLang]
})
```

Otherwise you can only use one preprocessor in a project.